scripts/cert-update.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+eval `ssh-agent`
+ssh-keyscan -H github.com >> ~/.ssh/known_hosts
```

Comment:
this adds the ssh fingerprint of github into the `~/.ssh/known_hosts`

https://explainshell.com/explain?cmd=ssh-keyscan+-H+github.com+%3E%3E+%7E%2F.ssh%2Fknown_hosts